### DISCUSSION TITLE
restrict users to only create repositories inside organisations

### DISCUSSION THREAD

**Discussion by WaiLoonTan** [community/community#22102] [Enterprise]
<p>Hi Github,</p>
<p>I am currently using Github Enterprise trial version : 2.14.4.</p>
<p>I want to restrict my users from creating repositories outside of organisations.</p>
<p>Meaning, i want them to create repositories inside their organisations they belong to.</p>
<p>How i can do that ? </p>
<p>Many thanks.</p>

**Comment by angela-crist** [comment, marked as answer]
<p>Hi <a class="mention" href="/wailoontan">@wailoontan</a>, we’re glad you’re here :grinning:</p>
<p>This is not a current feature. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by WaiLoonTan** [comment]
<p>Hi Angela,</p>
<p>Thanks ! Looking forward for the new feature !</p>
<p>Thanks. </p>

**Comment by rgarg-quotient** [comment]
<p>Any update on this feature. It’s a good to have security feature.</p>
